### PR TITLE
added waldo command

### PR DESCRIPTION
**Pull request by Nathan-Binkley**

### CODE REVIEW

File: bot/cogs/memes_cog.py [hunk 1]
```diff
@@ -25,6 +25,29 @@ async def bubblewrap(self, ctx):
             msg += '\n'
 
         await ctx.send(msg)
+        
+    @commands.command()
+    async def Waldo(self, ctx, size=max_waldo_grid_size):
```
Comment (Jay-Madden) [inline bot/cogs/memes_cog.py:30]: you forgot to copy this

Comment (Nathan-Binkley) [reply bot/cogs/memes_cog.py:31]: Don't worry
I'm just bad at code

File: bot/cogs/memes_cog.py [hunk 2]
```diff
@@ -12,6 +12,7 @@
 
 class MemesCog(commands.Cog):
 
+    max_waldo_grid_size = 100
```
Comment (Jay-Madden) [inline bot/cogs/memes_cog.py:15]: make this a module constant, so it goes above the class def and below the logger

File: bot/cogs/memes_cog.py [hunk 3]
```diff
@@ -26,6 +28,29 @@ async def bubblewrap(self, ctx):
...
+        msg = ''
+        count = 0
+        place = random.randint(0,size)
+
+        for i in range(size+1):
+            if i == place:
+                msg += '||WALDO|| '
+                count += 1
+            else:
+                msg += '||MALDO|| '
```
Comment (Jay-Madden) [inline bot/cogs/memes_cog.py:45]: what are your thoughts on randomizing the first letter of the non waldo ones, like `Maldo` `HALDO` `NALDO` etc

Comment (Nathan-Binkley) [reply bot/cogs/memes_cog.py:45]: I tried.

Didn't work. Spacing was off. BALDO has a different length than WALDO which has a different length than everything other than MALDO

Comment (Nathan-Binkley) [reply bot/cogs/memes_cog.py:45]: If you knew the proper length, it'd be incredibly easy to see it

Comment (Jay-Madden) [reply bot/cogs/memes_cog.py:45]: ahh gotcha, makes sense

**Pull request merged**